source/System.cpp

```diff
@@ -534,6 +535,14 @@ void System::UpdateSystem(const Set<System> &systems, const set<double> &neighbo
 
 
 
+void System::UpdateObjectVisibilities(const Ship *ship)
```

Comment:
same, if its smth we need its gotta be a ref (or you'd at least have to check if !ship)

Reply:
mhm